build/update-opa-version.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
 # Script to revendor OPA, Add and Commit changes if needed
 
+set +e
```

Comment:
Did you mean for this to be +e? +e will make the script NOT exit if subprocesses error.